lib/laa_crime_schemas/structs/amount.rb

```diff
@@ -5,6 +5,7 @@ module Structs
     class Amount < Base
       attribute :amount, Types::PenceSterling
       attribute :frequency, Types::PaymentFrequency
+      attribute :ownership_type, Types::OwnershipType
```

Comment:
Need to also add this to payments in apply

Reply:
I'd imagined ownership type would be mixed in to the parent object along side Amount.